logbook-api/src/main/java/org/zalando/logbook/RequestURI.java

```diff
@@ -16,29 +16,41 @@ private RequestURI() {
 
     }
 
-    enum Component {
+    static enum Component {
         SCHEME, AUTHORITY, PATH, QUERY
     }
 
-    static String reconstruct(final HttpRequest request) {
-        return reconstruct(request, EnumSet.allOf(Component.class));
+    public static String reconstruct(final HttpRequest request) {
```

Comment:
All methods can drop the public modified as well, right?

Reply:
Yes, forgot.